gbfs.md

```diff
@@ -192,20 +197,53 @@ stations          | Yes       | Array that contains one object per station in th
 \- capacity        | Optional  | Number of total docking points installed at this station, both available and unavailable
 
+### geofencing_zone_information.json
+All geofencing zones contained in this list are considered public (ie, can be shown on a map for public use). If there are private geofencing zones these should not be exposed here and their status should not be included in geofencing_zone_information.json.
+
+Field Name        | Required  | Defines
+------------------| --------- | ----------
+geofencing_zones   | Yes       | Array that contains one object per geofencing zone in the system as defined below
+\- geofencing_zone_id      | Yes       | Unique identifier of a geofencing zone. See [Field Definitions](#field-definitions) above for ID field requirements
+\- name            | Optional       | Public name of the geofencing zone
+\- lat             | Yes       | The latitude of the centroid of the geofencing zone. The field value must be a valid WGS 84 latitude in decimal degrees format. See: http://en.wikipedia.org/wiki/World_Geodetic_System, https://en.wikipedia.org/wiki/Decimal_degrees
+\- lon             | Yes       | The longitude of the centroid of the geofencing zone. The field value must be a valid WGS 84 longitude in decimal degrees format. See: http://en.wikipedia.org/wiki/World_Geodetic_System, https://en.wikipedia.org/wiki/Decimal_degrees
+\- region_id       | Optional  | ID of the region where geofencing zone is located (see [system_regions.json](#system_regionsjson))
+\- post_code       | Optional  | Postal code where the geofencing zone is located
+\- capacity        | Optional  | Total bike capacity approved for this geofencing zone, as approved by the city or the operator.
+\- zone_area	    | Yes	    | A valid GeoJSON MultiPolygon that describes the area where free bikes can be dropped off. 
+\- station_id      | Optional  | Unique identifier of the station this geofencing zone is attached to, if any (see station_information.json)
+
+
 ### station_status.json
 
 Field Name            | Required  | Defines
 --------------------- | ----------| ----------
 stations              | Yes       | Array that contains one object per station in the system as defined below
 \- station_id          | Yes       | Unique identifier of a station (see station_information.json)
 \- num_bikes_available | Yes       | Number of bikes available for rental
+\- num_bikes_available_types | Yes | Array that contains one entry per bike type as defined below (more types can be added)
```

Comment:
@dsgermain what you mean by array of entries? Each entry should be an object like `[{ 'ebike': 5 }, { 'mechanical': 3 }]` ? Can you give an example?

Reply:
@hkieferling Yes, your example is correct. It would be nice to have a base set (enum) of approved bike types which the vendors can extend if it doesn't fit their model.